js/adapt-contrib-xapi.js

```diff
@@ -1166,35 +1165,21 @@ define([
       });
     },
 
-    showErrorView: function() {
-      if (this.getConfig('_lrsFailureBehaviour') === 'ignore') {
-        return;
-      }
+    getGlobals: function() {
+      return Adapt &&
+        Adapt.course &&
+        Adapt.course.get('_globals') &&
+        Adapt.course.get('_globals')._extensions &&
+        Adapt.course.get('_globals')._extensions._xapi
+        || {};
```

Comment:
Instead of an empty object here, should the `confirm` value, etc. be defaulted?

Reply:
The only reason I didn't do that (it would be my preference) is that this has come up in the past, and the recommendation was always to not have hard-coded strings, even when they're just used as a fallback.

Reply:
If these attributes are missing from the JSON though this will result in displaying empty strings to the user.  

If you add in the defaults I'll approve.